lib/src/main/proto/bio/terra/tanagra/underlay/tables.proto

```diff
@@ -48,3 +48,18 @@ message ColumnId {
   // The name of the column.
   string column = 3;
 }
+
+// A filter specification to get a subset of the rows in a table.
+message TableFilter {
+  // A condition on a particular column
+  // TODO: allow multiple ColumnFilters and AND/OR'ing them together
+  ColumnFilter column_filter = 1;
+}
+
+// A building block for a TableFilter that defines a condition on a single column.
+message ColumnFilter {
+  // Pointer to the column.
+  ColumnId column = 1;
+  // Column value to filter on
+  string value = 2;
```

Comment:
nit: I think it may be clearer to model this as a oneof to indicate future extension (with just `string_value` for now). The filter here I think can be parallel to what is defined by the API for entity search: https://github.com/DataBiosphere/tanagra/blob/328febf90e7f8a5c31b7ab8258d5a1bcbdb0bcb2/api/src/main/resources/api/service_openapi.yaml#L316-L329

Reply:
Thanks for the pointer, I've not used protos before so glad to learn about `oneof`. Changed the `ColumnValue` to accommodate both strings and ints, to be consistent with the two currently supported column data types.